www/src/components/PrivateRoute.js

```diff
@@ -0,0 +1,8 @@
+const PrivateRoute = ({component: Component, authenticated, ...rest}) => {
+  return (
+    <Route {...rest} render={(props) => authenticated === true
+        ? <Component {...props} />
+        : <Redirect from={`${props.match.path}`} to={`/login`} />}
+    />
+  )
+}
```

Comment:
PR became larger than necessary in trying to handle state management. However this will be used later on with tools like the admin for content management.